crates/nu-command/src/date/now.rs

```diff
@@ -42,6 +42,11 @@ impl Command for DateNow {
                 example: r#"date now | format date "%Y-%m-%d %H:%M:%S""#,
                 result: None,
             },
+            Example {
+                description: "Get the current date and display it in RFC3339 format string.",
```

Comment:
```suggestion
                description: "Get the current date and format it according to the RFC3339 standard.",
```

Reply:
This one seems appropriate. Resolving both according to this verbiage.